templates/ai/consulting.html

```diff
@@ -0,0 +1,357 @@
+        objectives.
+      </p>
+      <p>
+        Together, we help deliver a five-day/five-step AI design sprint with your analytics and infrastructure teams. At the end of the engagement you will have a pattern for productive developer workstations, a machine learning infrastructure (in the cloud or on-premises), and AI applications delivering daily insights, powered by your data.
+      </p>
+      <p class="u-sv3">
+        With our structured program, we can approach any situation and provide the infrastructure and capabilities that your business needs. Proven best practices mean that we can take the guesswork and experimentation out of your AI adoption, and fast-track you to value without breaking your budget.
+      </p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <p class="u-align--center">OUR CONSULTING PARTNERS</p>
+      <ul id="cloud-partners" class="p-inline-images">
+        <li class="p-inline-images__item">
+          <img
+            class="p-inline-images__logo"
+            src="{{ ASSET_SERVER_URL }}7a581e36-Manceps_Logo_160.png"
+            alt="Manceps"
+          />
+        </li>
+        <li class="p-inline-images__item">
+          <img
+            class="p-inline-images__logo"
+            src="{{ ASSET_SERVER_URL }}99d8f79e-deepsense.png"
+            alt="Deepsense AI"
+          />
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>AI design sprint</h2>
+    <p class="p-heading--four u-sv3">
+      Produce tangible results in one week with agile AI technologies and
+      methodologies
+    </p>
+    <p>
+      The core of our one week engagements centers on the AI design sprint - an
+      introduction our AI technologies and methodologies. We will develop a
+      proof of concept (PoC) that solves a real business challenge for your
+      organization. The results of this sprint will serve as a foundation for
+      your AI innovations.
+    </p>
+    <p class="u-sv3">
+      The key outcomes that you can expect from this week are a baseline
+      architecture for iterative solution development – which includes a working
+      continuous integration pipeline – and a high level strategy that addresses
+      your AI transformation journey. Please note: this week typically relies on
+      having the right infrastructure in place to support the AI technology
+      stack.
+    </p>
+    <p class="p-heading--three u-sv3">
+      The AI design process in more detail
+    </p>
+  </div>
+  <div class="row">
+    <div class="col-6 u-hide--small u-align-center">
+      <img src="{{ ASSET_SERVER_URL }}5d8aa78b-1+Discovery.svg" alt="" width="335px"/>
+    </div>
+    <div class="col-6">
+      <p class="p-heading--four">1 - Discovery</p>
+      <p>
+        Following a consultative whiteboarding session to set the stage, partner data scientists and data engineers
```

Comment:
you don't have to do hard breaks in lines, it gets messy as you edit... you don't have to remove here, but in the future, don't bother